### PR TITLE
Fix security vulnerability for ActionView

### PR DESCRIPTION
**Pull request by rohandaxini**
# Description

Fix for [XSS Vulnerability in Action View tag helpers](https://github.com/ifmeorg/ifme/security/dependabot/69).

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ ruby '~> 2.6.10'
 
 gem 'puma'
 gem 'rack-rewrite'
-gem 'rails', '~> 6.0.3.3'
+gem 'rails', '~> 6.0.4.8'
```
Comment (julianguyen) [inline Gemfile:8]: Let's also update documentation in our wiki to this version of Rails!

**Review by julianguyen — APPROVED** [review]
Looks good to me!

File: Gemfile (hunk 1 above)
Comment (rohandaxini) [reply Gemfile:8]: Sure @julianguyen Once you'll merge this PR, ill immediately update the version number [here](https://github.com/ifmeorg/ifme/wiki/Installation). Is there any other place or other wiki where version number of Rails should be updated?

**Comment by rohandaxini** [comment]
@julianguyen Yay 🎉 
This [XSS Vulnerability](https://github.com/ifmeorg/ifme/security/dependabot/69) shows fixed now.

**Pull request merged**